agents/threadpool/hippo4j-agent-plugin/spring-plugins/spring-plugin-common/src/main/java/cn/hippo4j/agent/plugin/spring/common/conf/SpringBootConfig.java

```diff
@@ -22,22 +22,37 @@
 import java.util.Arrays;
 import java.util.List;
 
+/**
+ * Spring boot config
+ */
 public class SpringBootConfig {
 
+    /**
+     *  Spring
+     */
     public static class Spring {
 
+        /**
+         * Dynamic
+         */
         public static class Dynamic {
 
+            /**
+             * ThreadPool
+             */
             @SpringBootConfigNode(root = SpringBootConfig.class)
-            public static class Thread_Pool {
+            public static class ThreadPool {
```

Comment:
This cannot be changed here, and it will affect the use of functions. should be added to the ignore file.

Reply:
Please fix this problem as soon as possible. I am developing the Agent module recently, and I am worried that there will be conflicts in the future, which will make it difficult to merge.